content/tutorials/mapping.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Using SLAM for Map Building"
+date: 2021-03-20T17:15:01-07:00
+summary: "Learn how to create custom maps of your car's surroundings"
+difficulty: Intermediate
+duration: 0
+featured: true  # whether this is listed at / (must also be top 6 by weight)
+active: true    # whether this is listed at /tutorials/
+draft: false      # whether Hugo considers this a draft
+weight: 100
+---
+
+## By: Madison Doerr and Markus Schiffer
+
+### Introduction
+The MuSHr car operates in a mapped area to determine its position, orientation, and plans. We can create 
+custom 2D maps for our cars by using [ROS's slam](http://wiki.ros.org/slam_toolbox) and our car's laser scanner to 
+survey the area to be mapped.
+
+### Goal
+Create a custom map of an area by driving the car to survey the surroundings.
+
+### Requirements
+  - Complete the [quickstart](/tutorials/quickstart) tutorial
+  - Complete the [first steps](/tutorials/first_steps) tutorial
+  - A desktop/laptop that can ssh into the car
+
+### Ssh into the car
+Boot up the car and connect the controller. Ssh into the car on your computer, replacing the IP address with your robot IP when connected with the ROBOT_AP network, or using the Wifi connect from the [first steps](/tutorials/first_steps) tutorial.
+```bash
+$ ssh robot@10.42.0.1 -X
+```
+
+### Install gmapping
+In a terminal sshed into the car using the above steps, make sure gmapping is installed.
```

Comment:
Was this installed for you by default?

Reply:
Yeah, so this was installed by default, however, I could not get it to work without installing again and I think Markus had the same issue. Maybe we can switch this to sudo apt-update?